source/features/show-whitespace.tsx

```diff
@@ -76,5 +76,6 @@ void features.add(__filebasename, {
 		onNewComments,
 		onPrFileLoad
 	],
+	deduplicate: '.rgh-observing-whitespace',
```

Comment:
Won't this break `additionalListeners`? 

`.rgh-observing-whitespace` matches any observed line, which may still be on the page when this check runs.

I think we'll need an additional class probably on the top-most element that is replaced (`#pjax-container > .rgh-observing-whitespace:first-child` ? not sure)

Reply:
> Won't this break `additionalListeners`?

I do not think so its not excluded in the deduplication

Reply:
I think we had this discussion before. You're right 😃